packages/sdk-metrics/test/aggregator/Histogram.test.ts

```diff
@@ -81,6 +81,27 @@ describe('HistogramAggregator', () => {
         sum: -65,
       });
     });
+
+    it('with single bucket', function () {
+      const aggregator = new HistogramAggregator([0], true);
+      const prev = aggregator.createAccumulation([0, 0]);
+      prev.record(0);
+      prev.record(1);
+
+      const delta = aggregator.createAccumulation([1, 1]);
+      delta.record(2);
+      delta.record(11);
+
+      const expected = aggregator.createAccumulation([0, 0]);
+      // replay actions on prev
+      expected.record(0);
+      expected.record(1);
+      // replay actions on delta
+      expected.record(2);
+      expected.record(11);
+
+      assert.deepStrictEqual(aggregator.merge(prev, delta), expected);
```

Comment:
It might be better to explicitly initialize the `expected value, as we are verifying the actual structure of the accumulation.

```suggestion
      const expected = new HistogramAccumulation([0, 1], [], true, {
        buckets: {
          boundaries: [],
          counts: [4],
        },
        count: 4,
        sum: 14,
        hasMinMax: true,
        min: 0,
        max: 11,
      });
      assert.deepStrictEqual(aggregator.merge(prev, delta), expected);
```

Reply:
agreed, changed in [1641bd7](https://github.com/open-telemetry/opentelemetry-js/pull/4456/commits/1641bd70d95b2c93c28ccc68d02abbcc42fe8716) :slightly_smiling_face: